core/src/main/java/io/undertow/conduits/ChunkedStreamSinkConduit.java

```diff
@@ -214,12 +316,7 @@ public void truncateWrites() throws IOException {
 
     @Override
     public long write(final ByteBuffer[] srcs, final int offset, final int length) throws IOException {
```

Comment:
At the cost of some single-element array allocations, we could delegate the single-buffer implementation to this one, just like sslconduit